### ISSUE TITLE
Render to Cursive View?

### ISSUE THREAD

**Issue by matthiasbeyer** [lotabout/skim#238]
Hi,

would it technically be possible to use Cursive as a frontend for skim? So that I can render the skim selection into a Cursive View?

The idea behind this would be a way to integrate skim (as a library) into a Cursive-based application.

If yes, please consider this a feature request. I would like to help (discussion/design/code) to implement such a feature.

**Comment by LoricAndre** [comment]
We are migrating to ratatui, and I think the app itself can _technically_ be used as a Ratatui widget, but Cursive is not planned

**Comment by matthiasbeyer** [comment]
Yeah, that issue was written in a time where ratatui did not exist (at least in my perception). So I am totally fine now 😆
